src/style.css

```diff
@@ -8,10 +8,30 @@
   background: url(./images/star.svg) no-repeat center;
 }
 
-.stars__star.is-active {
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
   background: url(./images/star-active.svg) no-repeat center;
 }
 
-.stars__star:not(:first-child) {
-  margin-left: 4px;
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.stars--6 .stars__star:nth-child(-n + 6) {
+  background: url(./images/star-active.svg) no-repeat center;
 }
```

Comment:
combine all these selectors to prevent code duplicating

Reply:
Thanks for advice!